src/fallback.ts

```diff
@@ -135,7 +135,10 @@ export class GrpcClient {
   }
 
   loadProtoJSON(json: protobuf.INamespace, ignoreCache = false) {
-    const hash = objectHash(json).toString();
+    const hash = crypto
+      .createHash('md5')
```

Comment:
That works, but please don't use MD5 (it's safe here but normally we don't allow MD5 anywhere).  Use SHA1.

Reply:
For learning purpose, how do we determin which algorithm to use, beside `MD5`, I see that we have `SHA1`, `SHA2` and `SHA256`.

Reply:
Decide to use `objectHash`